package.json

```diff
@@ -32,8 +33,9 @@
     "deploy:kovan": "npx hardhat --config hardhat.kovan-staging-config.ts run scripts/kovan-deployment.ts --network kovan",
     "frontend": "cd frontend && npm run start",
     "node": "npx hardhat node --watch",
-    "test": "npx hardhat test ./test/unit/**/*.ts",
-    "test-integration": "npx hardhat --config hardhat.integration-test-config.ts test ./test/integration/**/*.ts"
+    "test": "npm run test:unit && npm run test:integration",
+    "test:unit": "TS_NODE_TRANSPILE_ONLY=1 npx hardhat test ./test/unit/*.ts",
```

Comment:
is there an explaination to what `TS_NODE_TRANSPILE_ONLY=1` does? curious.

Reply:
yep, when we run our tests, typescript typechecks everything while running the tests which is very expensive. This flags turns of typechecking off, and simply transpiles all the code from TS to JS when running tests.

see here: https://hardhat.org/guides/typescript.html#performance-optimizations